### DISCUSSION TITLE
OPDS. группировка по авторам

### DISCUSSION THREAD

**Discussion by plotn** [petrovvlad/freeLib#56] [Ideas]
И снова я и снова со "своей" флибустой. ОПДС отдает список авторов в виде - сначала по одной букве, потом все, что лежит "в этой букве". В силу огромности флибусты - например на букве К - 15026 авторов. На букве М - 10357. Это сильно затрудняет поиск. А вот в сериях у вас сделано "сначала одна буква, потом вторая" и т.д.

**Comment by petrovvlad** [comment]
Толь ко что проверил. Навигация по авторам и сериям организовано одинаково.

**Comment by plotn** [comment]
можете мне на почту написать?, я вам дам ссылку на свою библиотеку.
Моя:
plotnikovya
жмайл ком.
У меня на букве "У" действительно группирует как и серии, а вот на группе "М" - нет. Может быть есть какая то ошибка в коде группировки?

**Comment by petrovvlad** [comment]
Если количество авторов/серий начинающихся с набранных букв ≤ nMaxBooksPerPage, то отображаются авторы/серии целиком, если >, то отображаются варианты начала автора/серии с количеством букв на 1 больше.

**Comment by petrovvlad** [comment]
Поправил навигацию. Проверьте. Если вылезут ещё аномалии при навигации, пишите.

**Reply by plotn** [reply]
да, теперь нормально стало, спасибо!
